### PR TITLE
Add working docker-compose config & link in README

### LINKED ISSUE #22: Provide docker-compose configuration
I just spent an awful amount of time today trying to get the provided `pentacent/keila` running on a private server, mostly due to a messy PostgreSQL setup.

Maybe a `docker-compose.yml` in the likes of the sketch below could be useful? Would be lovely to being able to "just" run `docker-compose run -d app` and be done with the setup 😎 happy to help with it, too!

```YAML
version: "3"

services:
  app:
    image: pentacent/keila
    ...

  db:
    image: postgres:13.2
    ...
```

### PR DESCRIPTION
**Pull request by pascalwengerter**
Fixes #22

### CODE REVIEW

**Comment by pascalwengerter** [comment]
@wmnnd funnily enough heavily inspired by [papercups-io](https://github.com/papercups-io/papercups/blob/master/docker-compose.prod.yml) 😅

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ to SMTP.
 
 Keila is ready for you to send newsletters with!
 
-You can easily deploy it using the official Docker image `pentacent/keila`.
+You can easily deploy it using the official Docker image `pentacent/keila` or use the [Docker Compose config](ops/docker-compose.yml) in this repo.
```
Comment (wmnnd) [inline README.md:21]: I would suggest changing the wording. I see the docker-compose file as a way to quickly try Keila but not necessarily a proper deployment recommendation.

File: ops/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+version: "3"
+
+services:
+  app:
```
Comment (wmnnd) [inline ops/docker-compose.yml:4]: Okay, time to be super nitpicky 😂

Comment (wmnnd) [inline ops/docker-compose.yml:4]:
```suggestion
  keila:
```

File: ops/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    image: pentacent/keila
+    ports:
+      - "4000:4000"
+    command:
```
Comment (wmnnd) [inline ops/docker-compose.yml:8]: I suggest removing this and instead adding `CMD ["start"]` to the Dockerfile.

File: ops/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+version: "3"
+
+services:
+  app:
+    image: pentacent/keila
```
Comment (wmnnd) [inline ops/docker-compose.yml:5]:
```suggestion
    image: pentacent/keila:latest
```

File: ops/docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+      - "4000:4000"
+    command:
+      - "start"
+    depends_on:
+      - db
+    build:
+      context: .
+    environment:
+      SECRET_KEY_BASE: "${YOUR_SECRET_KEY}"
+      DB_URL: "ecto://postgres:postgres@db/postgres"
```
Comment (wmnnd) [inline ops/docker-compose.yml:16]:
```suggestion
      DB_URL: "postgres://postgres:postgres@postgres/postgres"
```
<- This is the best URL ever!

File: ops/docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+      URL_HOST: "${YOUR_URL_HOST}"
+      MAILER_SMTP_HOST: "${YOUR_MAILER_SMTP_HOST}"
+      MAILER_SMTP_USER: "${YOUR_MAILER_SMTP_USER}"
+      MAILER_SMTP_PASSWORD: "${YOUR_MAILER_SMTP_PASSWORD}"
+
+  db:
```
Comment (wmnnd) [inline ops/docker-compose.yml:22]:
```suggestion
  postgres:
```

File: ops/docker-compose.yml [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    image: postgres:alpine
+    container_name: database
```
Comment (wmnnd) [inline ops/docker-compose.yml:24]:
```suggestion
```

File: ops/docker-compose.yml [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+    ports:
```
Comment (wmnnd) [inline ops/docker-compose.yml:25]: Ports for the DB don’t have to be exposed.
```suggestion
```

File: ops/docker-compose.yml [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+      - "5432:5432"
```
Comment (wmnnd) [inline ops/docker-compose.yml:26]:
```suggestion
```

**Review by wmnnd — CHANGES_REQUESTED** [review]
Thank you for working on the `docker-compose` file! I tried it with the latest release (`0.2.2`) and it seems like it’s finally working out of the box 😅

I think we’re mostly good to go like this, I just have some small nitpicks and some suggestions to make the docker-compose file a bit more minimal.

File: README.md (hunk 1 above)
Comment (pascalwengerter) [reply README.md:21]: Got it, will adjust the wording accordingly!

File: ops/docker-compose.yml (hunk 3 above)
Comment (pascalwengerter) [reply ops/docker-compose.yml:8]: Do you suggest adding/publishing a `dev` Dockerfile, or changing the official image?

File: ops/docker-compose.yml (hunk 5 above)
Comment (pascalwengerter) [reply ops/docker-compose.yml:16]: Naming is important so people know _exactly_ what you're talking about, I guess. It doesn't get any more postgres-y than this one, yeah 😅

File: ops/docker-compose.yml (hunk 3 above)
Comment (wmnnd) [reply ops/docker-compose.yml:8]: I would suggest adding this in the actual `Dockerfile`; I don’t think a dockerized dev environment is something we need right now :smiley:

Comment (wmnnd) [reply ops/docker-compose.yml:8]: I’ve already added `CMD` to the Dockerfile now :partying_face:

**Review by wmnnd — APPROVED** [review]
Thank you, this should make it much easier for new people to give Keila a try! :partying_face:

**Pull request merged**